### PR TITLE
customer home page header and filters

### PR DESCRIPTION
**Pull request by waed-hamza**
# Customer home page 

# header:
1. App name.
2. Search bar.
3. Two buttons : customer and seller.
4. Cart Icon.

# Filters:
1. According to price.
2. According to category.
3. Display items as list.
4. Display items as grid.

# How design looks like:
image

### CODE REVIEW

File: Assets/css/customer.css [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+}
+
+
+/* Header */
+header{
+    position: fixed;
+}
+
+.header-div{
+    width: 1490px;
```
Comment (Rahaf-96) [inline Assets/css/customer.css:24]: insted of giving the `header-div` fixed width, give it `100vw` to make it responsive on all screens.

**Comment by Rahaf-96** [comment]
move `images` folder into the `assets` folder

File: html/customer.html [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://use.fontawesome.com/23bea62652.js"></script>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Overpass:wght@200&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Quicksand&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="/Assets/css/customer.css">
+    <title>Document</title>
```
Comment (Rahaf-96) [inline html/customer.html:11]: don't forget to change the page title.

File: Assets/css/customer.css (hunk 1 above)
Comment (waed-hamza) [reply Assets/css/customer.css:24]: resolved

File: html/customer.html (hunk 2 above)
Comment (waed-hamza) [reply html/customer.html:11]: Ok, and I added icon to the title

**Pull request merged**